keyboards/handwired/videowriter/keymaps/default/keymap.c

```diff
@@ -0,0 +1,131 @@
+/* Copyright 2019 quartz64
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE,
+    _FN1,
+    _FN2
+};
+
+// Defines the keycodes used by our macros in process_record_user
+/*
+enum custom_keycodes {
+    QMKBEST = SAFE_RANGE,
+    QMKURL
+};
+ */
+
```

Comment:
You can delete this block.

Reply:
Can I leave it here commented out so it can be used as a template for creating a macros? If it doesn't violate any formatting/coding conventions.